app/controllers/admin/sessions_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Admin::SessionsController < AdminApplicationController
+  before_action :authenticate_user!, only: %i[logout]
+  before_action :prevent_access_for_authenticated_user!, only: %i[login]
+
+  def login
+    if request.post?
+      login = Login.authenticate(params[:email], params[:password])
```

Comment:
TBD: I see another login collection for the admins.

Reply:
Shared user collection? Or should that be separate as well?

Reply:
Just realized we don't really need a User collection in this context. Will update.